### PR TITLE
feat: enable locally hosted models in Migration example

### PR DESCRIPTION
**Pull request by larsmoa**
Add support for locally hosted models to Cognite3DViewer and Cognite3DModel. 

Note that locally hosted models have several limitations due to lack of support from CDF that many Reveal features rely on.

This feature is meant for use in development only, and is not meant to be used in production. The toggle to enable this is not exported in the library and hence hidden to end users.

This feature is now enabled in the Migration example, which allows you to visit e.g. https://localhost:8080/Migration?modelUrl=/primitives to open the default test model:

image

### CODE REVIEW

File: viewer/core/src/public/migration/Cognite3DModel.ts [hunk 1]
```diff
@@ -14,11 +14,10 @@ import { trackError } from '../../utilities/metrics';
 
 import { SupportedModelTypes, CadModelMetadata } from '../types';
 import { callActionWithIndicesAsync } from '../../utilities/callActionWithIndicesAsync';
-import { CogniteClientNodeIdAndTreeIndexMapper } from '../../utilities/networking/CogniteClientNodeIdAndTreeIndexMapper';
 import { NodeCollectionBase } from '../../datamodels/cad/styling';
 import { NodeAppearance } from '../../datamodels/cad';
 import { NodeTransformProvider } from '../../datamodels/cad/styling/NodeTransformProvider';
-import assert from 'assert';
+import { NodesApiClient } from '../../../../packages/nodes-api';
```
Comment (christjt) [inline viewer/core/src/public/migration/Cognite3DModel.ts:20]: wrong import, should be ```@reveal/nodes-api```

File: viewer/packages/nodes-api/src/NodesCdfClient.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  async determineTreeIndexAndSubtreeSizesByNodeIds(
+    modelId: CogniteInternalId,
+    revisionId: CogniteInternalId,
+    nodeIds: CogniteInternalId[]
+  ): Promise<NodeTreeIndexAndSubtreeSize[]> {
+    const requests = nodeIds.map(id => ({ id }));
+    const nodes = await this._client.revisions3D.retrieve3DNodes(modelId, revisionId, requests);
+    return nodes.map(n => {
```
Comment (christjt) [inline viewer/packages/nodes-api/src/NodesCdfClient.ts:54]: rename `n` to `node`

**Review by christjt — CHANGES_REQUESTED** [review]

**Comment by larsmoa** [comment]
Comments fixed @christjt. PS! I love the 'Suggestion'-button for small changes.... ;)

**Comment by christjt** [comment]
> Comments fixed @christjt. PS! I love the 'Suggestion'-button for small changes.... ;)

I see your point, but sometimes I am not exactly sure what the syntax is. And in those cases it is easier to do in an IDE.

**Review by christjt — APPROVED** [review]

**Pull request merged**